src/platform/datapath_winuser.c

```diff
@@ -967,9 +967,11 @@ CxPlatDataPathInitialize(
         if (Datapath != NULL) {
             for (uint16_t i = 0; i < Datapath->ProcCount; i++) {
                 if (Datapath->Processors[i].IOCP) {
+#pragma prefast(suppress:6001, "SAL can't track processors allocation")
```

Comment:
I don't like having to do this. I don't understand why SAL is doing this. Can you add a loop after zeroing above that each IOCP is null, and wrap it in `#if defined(_PREFAST_) ... #endif`?

Reply:
That doesn't work, the analysis error still occurs even if I do that.